proxy/core/src/main/scala/io/cloudstate/proxy/eventing/EventingManager.scala

```diff
@@ -83,14 +77,14 @@ object EventingManager {
       else List(EventMapping(entity, endpoints))
     }
 
-  def createSupport(eventConfig: Config)(implicit materializer: ActorMaterializer): Option[EventingSupport] =
+  def createSupport(eventConfig: Config)(implicit system: ActorSystem): Option[EventingSupport] =
```

Comment:
@sean-walsh Have you verified that whatever was passed in as `materializer` here originally was a system-wide materializer?

Reply:
It wouldn't have been, since there was no system-wide one prior to 2.6. The guidance I got and read about was to use the shared system one whenever possible. The implicit ActorSystem puts that one in scope. Am I missing something? I do think I need to take another look at using shutdown() where the shared materializer is being used, such as in this case.

Reply:
@sean-walsh Yes there was system-wide materializers in previous versions. See this: https://doc.akka.io/api/akka/2.5.4/akka/stream/ActorMaterializer$.html

No, you shouldn't have to modify any `shutdown()`s, what I am asking is that no lifecycles should change as a part of the migration itself. I'd like to have all Materializers in this PRs being bound to the same lifecycles as they used to be, and then we can look into changing that later, but it is not a core concern of this upgrade. Does that make sense?

Reply:
Yes it does, thanks.

Reply:
The materializer that was passed in before was a system wide materializer - but not managed by Akka, it was managed by Cloudstate. Generally, there is no reason to create more than one materializer, it's a small but not insignificant overhead if each actor creates one.

Reply:
@jroper I think the main reason for binding a materializer to an actor is to make sure that the streams do not survive the actor.

Reply:
@viktorklang @jroper So put it back and pass in the system materializer as before or leave it as is?

Reply:
@sean-walsh you'll have to check the callsites first to figure out what the appropriate course of action is which preserves the original lifecycle.

Reply:
In this case a new ActorMaterializer is instantiated in CloudStateProxyMain and passed down the line implicitly, so my change was just to use the shared one. I'll look at the other(s) cases also.

Reply:
@sean-walsh What matters is if it is a system-bound one or if it is an actor-context one.

Reply:
I looked at the API, it is a completely separate materializer, neither the shared system one or one created with an actor context. This is created with the system rather than an actor context.

Reply:
@sean-walsh Then it is system bound, so keeping to a system bound is fine.

Reply:
Thanks.